peerconnection_go_test.go

```diff
@@ -279,44 +224,6 @@ func TestPeerConnection_GetConfiguration(t *testing.T) {
 // 	assert.Nil(t, err)
 // }
 
-// TODO Fix this test
-const minimalOffer = `v=0
```

Comment:
The JavaScript WebRTC API doesn't consider this SDP valid and was returning the following error:

```
InvalidAccessError: Failed to set remote offer sdp: The m= section:video is invalid. RTCP-MUX is not enabled when it is required.
```

I updated the SDP in order to fix the issue.

Reply:
Is `TODO Fix this test` still relevant? Is there something left to do here?